Never pass -I/usr/include to a compiler.  It does bad things.

Kerberos RT #3011

--- krb5-1.4.2.orig/src/krb5-config.in
+++ krb5-1.4.2/src/krb5-config.in
@@ -166,7 +166,11 @@
 fi
 
 if test -n "$do_cflags"; then
-    echo "-I${includedir}"
+    if test x"$includedir" != x"/usr/include" ; then
+        echo "-I${includedir}"
+    else
+        echo ''
+    fi
 fi
 
 
